client/my-sites/checkout/composite-checkout/components/wp-checkout.tsx

```diff
@@ -177,26 +174,6 @@ export default function WPCheckout( {
 	// always visible. It is not a step and its visibility is managed manually.
 	const [ isSummaryVisible, setIsSummaryVisible ] = useState( false );
 
-	// The "Order review" step is not managed by Composite Checkout and is shown/hidden manually.
-	// If the page includes a 'order-review=true' query string, then start with
-	// the order review step visible.
-	const [ isOrderReviewActive, setIsOrderReviewActive ] = useState( () => {
-		try {
-			const shouldInitOrderReviewStepActive =
-				window?.location?.search.includes( 'order-review=true' ) ?? false;
-			if ( shouldInitOrderReviewStepActive ) {
```

Comment:
Since we're removing this, we may as well remove the code that adds this query string. https://github.com/Automattic/wp-calypso/blob/3a687b374a93b2e72843d4426a8f9953dddded9e/client/my-sites/current-site/stale-cart-items-notice.js#L55

Reply:
I will get it in this PR 👍